pocs/tests/test_pocs.py

```diff
@@ -104,6 +110,25 @@ def pocs_with_dome(config_with_simulated_dome, db_type, dome):
     pocs.power_down()
 
 
+@pytest.fixture(scope='function')
+def pocs_with_mount(config_with_simulated_mount, db_type, mount):
```

Comment:
This should just be `pocs` rather than `pocs_with_mount` as having a mount would be the default. Need to change all instances below as well.

Reply:
> This should just be `pocs` rather than `pocs_with_mount` as having a mount would be the default. Need to change all instances below as well.

I have added pocs_with_mount because `test_power_down_while_running` requires to have `mount` but not `dome`. So, `pocs_with_mount` is used to have a `mount` only without the `dome`.

Reply:
> This should just be `pocs` rather than `pocs_with_mount` as having a mount would be the default. Need to change all instances below as well.

I have added pocs_with_mount because `test_power_down_while_running` requires to have `mount` but not `dome`. So, `pocs_with_mount` is used to have a `mount` only without the `dome`.